sign_up.php

```diff
@@ -114,26 +118,50 @@ function check_states($st_list, $data)
 <html>
 	<head>
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<title>Registration Form</title>
+		<title>Sign Up</title>
```

Comment:
Either logged IN or not in both cases title is Sign Up.
But after login its wrong.